pkg/pillar/hypervisor/kubevirt.go

```diff
@@ -1218,3 +1219,77 @@ func getConfig(ctx *kubevirtContext) error {
 	}
 	return nil
 }
+
+// Register the PCI address with Kubevirt
+// Refer https://kubevirt.io/user-guide/virtual_machines/host-devices/#host-preparation-for-pci-passthrough
+func registerWithKV(kvClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance, pciAssignments []pciDevice) error {
+
+	devices := make([]v1.HostDevice, len(pciAssignments))
+	// Define the KubeVirt resource's name and namespace
+	kubeVirtName := "kubevirt"
+	kubeVirtNamespace := "kubevirt"
+
+	// Retrieve the KubeVirt resource
+	kubeVirt, err := kvClient.KubeVirt(kubeVirtNamespace).Get(kubeVirtName, &metav1.GetOptions{})
+	if err != nil {
+		return logError("can't fetch the PCI device info from kubevirt %v", err)
+	}
+
+	pciHostDevs := kubeVirt.Spec.Configuration.PermittedHostDevices.PciHostDevices
+
+	for _, pa := range pciAssignments {
```

Comment:
i have this code for direct attach code for ethernet, wondering if we have conflict here:
```
+++ b/pkg/pillar/cmd/domainmgr/domainmgr.go
@@ -1982,6 +1982,10 @@ func reserveAdapters(ctx *domainContext, config types.DomainConfig) *types.Error
            if ibp == nil {
                continue
            }
+           if ctx.hvTypeKube && ibp.Type == types.IoNetEth {
+               log.Noticef("reserveAdapters: ethernet io, skip reserve")
+               continue
+           }
            log.Noticef("reserveAdapters processing adapter %d %s phylabel %s",
                adapter.Type, adapter.Name, ibp.Phylabel)
            if ibp.AssignmentGroup == "" {
```

Reply:
There is no conflict, in fact your code helps to filter out network devices from the list